Dockerfile

```diff
@@ -5,5 +5,7 @@ ADD ./templates /etc/confd/templates
 
 VOLUME ["/app/gogs/custom/conf"]
 
+ENV GOGS_SERVICE_ENABLE_REVERSE_PROXY_AUTHENTICATION=false GOGS_SERVICE_ENABLE_REVERSE_PROXY_AUTO_REGISTRATION=false GOGS_SERVICE_DISABLE_REGISTRATION=false GOGS_SERVICE_SHOW_REGISTRATION_BUTTON=true
```

Comment:
The reason why doing it the way I did is recommended, is because docker creates a layer each time you have a "ENV".